core/source/org/libreoffice/ide/eclipse/core/wizards/NewUnoProjectWizard.java

```diff
@@ -371,10 +371,8 @@ protected IStatus run(IProgressMonitor pMonitor) {
                 if (!language.getName().contains("Python")) {
                     mServiceSet.mProject = prj;
                     mServiceSet.doFinish(pMonitor, mActivePage);
-
-                    UnoidlProjectHelper.setProjectBuilders(prj);
-
                 }
+                UnoidlProjectHelper.setProjectBuilders(prj);
```

Comment:
This allows that in the '.project' file located under the Projct folder. The following set of line is added:-
    <buildCommand>
        <name>org.libreoffice.ide.eclipse.core.types</name>
        <arguments>
        </arguments>
    </buildCommand>